Docs/source/Hardware/Boards.rst

```diff
@@ -427,15 +541,593 @@ the +5V input and corresponding grounds are already isolated from the +12V line.
+          current quantities to be measured over to the BPS Amperes Board.   
+        * Justification for selection of specific part: This part is used because it provides a more
+          accurate way to measure current, compared to previous methods like the Hall-effect. While this 
+          part is not galvanically isolated, the BPS Amperes Board is able to isolate the current measurement 
+          before it's sent to the BPS Leader Board. Thus, all together, this method using a shunt resistor provides 
+          an accurate way to provide current and temperature measurements while maintaining the power isolation in 
+          the BPS Leader Board.  
+        * `Datasheet link <https://www.vishay.com/docs/31094/wsbm8518.pdf>`__
+    * Thermistor    
+        * Description: This part is used to measure temperature. 
+        * Why is it necessary: This part helps the shunt resistor make an accurate temperature measurement. 
+        * Justification for selection of specific part: This part is used because it meets the requirements 
+          of the AS on the BPS Amperes Board and this part fits on the board more easily due to its Surface Mount 0805 size. 
+        * `Datasheet link <https://www.mouser.com/datasheet/2/619/smd_0805_v_e-1543794.pdf>`__
+
+.. figure:: ../_static/ShuntBrdSch.png
+    :align: center
+
+    BPS Shunt Resistor Board Schematic
+
+**Dimensions: 37.592mm by 32.512mm**
+
+Requirements/Constraints:  
+    * This board is required to fit in the molded enclosure on the shunt resistor.
+      This molded enclosure is 40.1 ± 0.2 mm by 35.1 ± 0.2 mm (in inches, it is 1.580 ± 0.009 in by 1.380 ± 0.008 in).  
+    * The headers and mounting holes on this board must align with that of the molded enclosure of the shunt resistor.  
+
+Design Choices:    
+    * This board had dimensions just slightly smaller than that of the molded enclosure on the shunt resistor. 
+      This prevented the board from moving around too much within the molded enclosure and made placement of 
+      circuit parts easier.  
+
+.. figure:: ../_static/ShuntBrdLayout.png
+    :align: center
+
+    BPS Shunt Resistor Board Layout
+
+.. figure:: ../_static/ShuntBrdRenderFront.png
+    :align: center
+
+    Front of BPS Shunt Resistor Board Render
+
+.. figure:: ../_static/ShuntBrdRenderBack.png
+    :align: center
+
+    Back of BPS Shunt Resistor Board Render
+
+
+BPS Test Board
+==============
+
+Overview
+^^^^^^^^
+`GitHub Link <https://github.com/lhr-solar/BPS-TestPCB>`__
+
+`BOM Link <https://www.mouser.com/ProjectManager/ProjectDetail.aspx?AccessID=fb0d7a5641>`__ 
+
+Brief Description/Purpose:
+    The purpose of this board is to emulate the battery modules for minion board testing  
+
+Context
+^^^^^^^
+**Location of the Board:**  it's tool for testing the minion boards so it doesn’t have an exact location in the car 
+
+**List of I/O and Connections:**
+    
+    * +5V
```

Comment:
This has weird formatting when it is built. I would add "Power" in front of it because it might be due to the "+" being right after the "*".

Reply:
okay will do